File: Algorithms/yourname.github/Linearsearch.c
#include<stdio.h>
int main()
{
  int n,key,i;
  int a[n];
  printf("Enter n\n");
  scanf("%d",&n);
  printf("Enter the array values\n");
  for (i=0;i<n;i++)
  {
    scanf("%d",&a[i]);
  }
  printf("Enter the value to be searched:");
  scanf("%d",&key);
  for (i=0;i<n;i++)
  {
    if(a[i]==key)
    {
      printf("index of key is %d",i);
      break;
    }
  }
  return 0;
}

File: Programming_Languages/Python/Algorithm/Linear_Search.py
def Linear_Search(arr, target):
    for i in arr:
        if target==i:
            return True
    return False

arr,target = (2, 25, 29, 92, 39, 90, 3, 10, 79, 57),int(input("Enter Target to be searched : "))
# Custom Array input
# arr,target = list(map(int,input('Enter numbers : ').split())),int(input("Enter Target to be searched : "))
Conclusion = Linear_Search(arr, target)
print("Element Found") if Conclusion==True else print("Element Not Found")